src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -62,14 +62,15 @@ public class OtlpExporterOptions
 #endif
 
         /// <summary>
-        /// Gets or sets optional headers for the connection.
+        /// Gets or sets optional headers for the connection. Refer to the <a href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specifying-headers-via-environment-variables">
+        /// specification</a> for information on the expected format for Headers.
         /// </summary>
         public string Headers { get; set; }
 
         /// <summary>
         /// Gets or sets the max waiting time (in milliseconds) for the backend to process each span.
         /// </summary>
-        public int? TimeOut { get; set; }
+        public int? TimeoutMilliseconds { get; set; }
```

Comment:
Since we're changing the name anyway, is there a reason to use `int?` instead of `TimeSpan?` for this?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/BatchExportProcessorOptions.cs#L30 We have so far used int everywhere, so keep it consistent here?

Reply:
Okay, so can we follow best practices for .NET and make this a `Timestamp` since it represents a duration of time?

Reply:
Okay, if we're already using int then I guess let's stay consistent.